project-base/docker/conf/docker-compose.yml.dist

```diff
@@ -104,6 +104,8 @@ services:
         container_name: shopsys-framework-adminer
         ports:
             - "1100:8080"
+        environment:
```

Comment:
When we are "touching" this topic, I would also update adminer to the latest (adminer:latest) version so it is always up to date.

Reply:
changed to latest